incuna-sass/mixins/_em-scale.sass

```diff
@@ -1,20 +1,21 @@
 // Mixin to define a font size scale to allow easier use of ems
 
 // Allows for as many different breakpoints and font sizes as needed
-// Only $font-base is required, mixin can be used without any 
-// $breakpoint-sizes-list arguments, but at least one breakpoint and 
-// font-size change should be added to make the mixin worth using
+// $font-base is required, also at least one breakpoint and 
+// font-size change must be added for the mixin to work
 
 // The mixin is designed mobile first, so $font-base should be your
-// mobile font size and $breakpoint-sizes-list should start with your 
+// mobile font size and $breakpoints-map should start with your 
 // smallest breakpoint and first font-size change, then continue in 
 // sequence from there
 
+// $breakpoints-map should be structured as $breakpoint: $font-size
+
 // Example usage:
 
 // .thing
-//     @include em-scale(14px, (('screen and (min-width: 768px)', 16px), 
-//          ('screen and (min-width: 1025px)', 20px)))
+//     @include em-scale(14px, ('screen and (min-width: 768px)': 16px, 
+//          'screen and (min-width: 1025px)': 20px))
```

Comment:
Yeah, which is always appreciated =) but since examples can be copied, it may produce confusion that it won't work: the user may think the issue is with the mixin because the error output isn't very informative
```sass
@mixin em-scale
@include em-scale(14px, ('screen and (min-width: 768px)': 16px,
    'screen and (min-width: 1025px)': 20px))
```
```
Error: Invalid CSS after "... 768px)': 16px,": expected expression (e.g. 1px, bold), was "{"
        on line 2 of test.sass
        >> , ('screen and (min-width: 768px)': 16px, {
   ------------------------------------------^
```